webapp/src/containers/WalletPage/components/WalletTxnsNew/index.tsx

```diff
@@ -183,58 +223,88 @@ const WalletTxns: React.FunctionComponent<WalletTxnsProps> = (
                     </td>
-                  ) : (
-                    <td className={`${styles.txid__na}`}>
-                      {I18n.t('containers.wallet.walletPage.txidNotApplicable')}
-                    </td>
-                  )}
-                </tr>
-              ))}
+                    {item.unit ? (
+                      <td>
+                        <div className={styles.amount}>
+                          {getAmountInSelectedUnit(
+                            item.amount,
+                            item.unit,
+                            item.unit
+                          )}
+                          &nbsp;
+                          <span className={styles.unit}>{item.unit}</span>
+                        </div>
+                      </td>
+                    ) : (
+                      <td
+                        className={`text-right ${getAmountClass(
+                          item.category
+                        )}`}
+                      >
+                        <div
+                          className={getPoolSwapClass(
+                            item.category,
+                            item.amounts[0].value
+                          )}
+                        >
+                          {`${numberWithCommas(item.amounts[0].value)} ${
+                            item.amounts[0].symbolKey
+                          }`}
+                        </div>
+                        {item.amounts[1] && (
+                          <div>
+                            {`${numberWithCommas(item.amounts[1].value)} ${
+                              item.amounts[1].symbolKey
+                            }`}
+                          </div>
+                        )}
+                      </td>
+                    )}
+                    {item.txid ? (
+                      <td>
+                        <div
+                          className={`${styles.txidvalue} ${styles.copyIcon}`}
+                        >
+                          <ValueLi
+                            value={item.txid}
+                            copyable={true}
+                            textLimit={textLimit}
+                          />
+                        </div>
+                      </td>
+                    ) : (
+                      <td className={`${styles.txid__na}`}>
+                        {I18n.t(
+                          'containers.wallet.walletPage.txidNotApplicable'
+                        )}
+                      </td>
+                    )}
+                  </tr>
+                ))}
             </tbody>
           </Table>
         </Card>
         <Pagination
-          label={I18n.t('containers.wallet.walletPage.paginationRange', {
-            to,
-            total,
-            from,
-          })}
+          // label={I18n.t('containers.wallet.walletPage.paginationRange', {
+          //   to,
+          //   total,
+          //   from,
+          // })}
```

Comment:
why is this label removed

Reply:
That was removed when We were trying to implement consolidating the txns from two calls. Because of pagination issue. But now include it again as started to get all data from listaccounthistory. I will again include it.
Thanks!